src/ssa_tracker.rs

```diff
@@ -11,6 +11,8 @@ pub struct SSATracker {
     /// TODO "highest" and "names" could be shared by copies of SSATracker...
     highest: BTreeMap<u64, u64>,
     names: BTreeMap<u64, String>,
+    /// Types of variables that are NOT 256 bit bitvectors.
+    types: HashMap<u64, String>,
```

Comment:
Why HashMap instead of BTreeMap?

Reply:
Because it's faster (is it?) and we don't need the ordering.